result/lldpneighbors.yaml

```diff
@@ -0,0 +1,155 @@
+            contain neighbor information for all configured LLDP interfaces.
+          type: array
+          items:
+            type: string
+          x-constraint:
+            - "/components/schemas/Device.Lldp/properties/name"
+        id_filters:
+          description: >-
+            Specify which neighbors to return. If the list is empty or missing
+            then all neighbors will be returned.
+          type: array
+          items:
+            type: string
+    
+    LldpNeighbors.State:
+      description: >-
+        LLDP neighbor information.
+      type: object
+      properties:
+        lldp_name:
+          description: >-
+            The name of a LLDP interface.
+          type: string
+        system_name:
+          description: >-
+            The system name field shall contain an alpha-numeric string that 
+            indicates the system's administratively assigned name. The system name 
+            should be the system's fully qualified domain name. If implementations 
+            support IETF RFC 3418, the sysName object should be used for this field.
+          type: string
+        system_description:
+          description: >-
+            The system description field shall contain an alpha-numeric string that 
+            is the textual description of the network entity. The system description 
+            should include the full name and version identification of the system's 
+            hardware type, software operating system, and networking software. If 
+            implementations support IETF RFC 3418, the sysDescr object should be used 
+            for this field.
+          type: string
+        chassis_id:
+          description: >-
+            The Chassis ID is a mandatory TLV which identifies the chassis component of 
+            the endpoint identifier associated with the transmitting LLDP agent.
+          type: string
+        chassis_id_type:
+          description: >-
+            This field identifies the format and source of the chassis identifier string. 
+            It is an enumerator defined by the LldpChassisIdSubtype object from IEEE 802.1AB 
+            MIB.
+          type: string
+          enum: [port_component, network_address,chassis_component,mac_address,interface_name,local,interface_alias]
+        id:
+          description: >-
+            System generated identifier for the neighbor on the interface.
+          type: string
+        age:
+          description: >-
+            Age since discovery in seconds.
+          type: integer
+          format: int64
+        last_update:
+          description: >-
+            Seconds since last update received.
+          type: integer
+          format: int64
+        ttl:
+          description: >-
+            The time-to-live (TTL) is a mandatory TLV which indicates how long information from the neighbor 
+            should be considered valid.
```

Comment:
If ttl is in seconds here, then mention seconds as unit.

Reply:
added seconds